catkit2/services/allied_vision_camera/allied_vision_camera.py

```diff
@@ -231,6 +248,19 @@ def make_property_helper(name, read_only=False):
             else:
                 self.make_property(name, lambda: getattr(self, name), lambda val: setattr(self, name, val))
 
+        # Set device values from config file (set width and height before offsets)
+        offset_x = self.config.get('offset_x', 0)
+        offset_y = self.config.get('offset_y', 0)
+
+        self.width = self.config.get('width', self.sensor_width - offset_x)
+        self.height = self.config.get('height', self.sensor_height - offset_y)
+        self.offset_x = offset_x
+        self.offset_y = offset_y
```

Comment:
Could you read from the configfile right here, like is done in the FLIR service? It would still set the offsets after the width and height, but we could delete the two lines in 252 and 253.

```suggestion
        self.offset_x = self.config.get('offset_x', 0)
        self.offset_y = self.config.get('offset_y', 0)
```

Reply:
I'm not changing this because of the fact the assignement of width and height needs the offsets in what was coded previously.